conf/distro/rpb-sota.conf

```diff
@@ -0,0 +1,11 @@
+require conf/distro/include/rpb.inc
```

Comment:
why don't we include:
require conf/distro/sota.conf.inc

like it seems to be recommended in meta-updater? their .inc file is doing a few other things that we aren't doing here, for example adding usrmerge

Reply:
Thanks for pointing it out. It should be included. I have updated the patch.